support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -223,14 +218,15 @@ function getClickHandler(props: PropTypes, isCustomPrb: boolean) {
 			props.useLocalCurrency,
 		);
 
-		if (
-			isCustomPrb &&
-			isValid &&
-			props.paymentRequestObject.status === 'AVAILABLE'
-		) {
-			props.paymentRequestObject.paymentRequest.show();
-		} else if (!isCustomPrb && !isValid) {
+		if (!isValid) {
 			event.preventDefault();
```

Comment:
If it's invalid we call `event.preventDefault` and then return. This is needed to prevent the stripe elements payment request button from opening the card popup. It's not necessary for a custom payment request button but I think it's fine to call it in that case too anyway